pkg/trace/test/testsuite/testdata/clientstats.go

```diff
@@ -3,8 +3,25 @@ package testdata
 import (
 	"github.com/DataDog/datadog-agent/pkg/trace/pb"
 	"github.com/DataDog/datadog-agent/pkg/trace/stats"
+	"github.com/DataDog/datadog-agent/pkg/trace/stats/quantile"
+
+	"github.com/DataDog/sketches-go/ddsketch"
+	"github.com/DataDog/sketches-go/ddsketch/mapping"
+	"github.com/DataDog/sketches-go/ddsketch/store"
+	"github.com/golang/protobuf/proto"
 )
 
+func getEmptyDDSketch() []byte {
```

Comment:
In this file, we are only testing if distributions are created with the correct keys. That's why we use empty sketches. We test the actual conversion in `conversion_test.go`